guides/common/modules/proc_configuring-alternate-content-sources.adoc

```diff
@@ -0,0 +1,54 @@
+[id="Configuring_Alternate_Content_Sources_{context}"]
+= Configuring Alternate Content Sources
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Alternate Content Sources*.
+. Click *Add Source* and set the *Source type* as *Custom*.
+. Select the *Content type*.
+. In the *Name* field, enter a name for the alternate content source.
+. Optional: In the the *Description* field, provide a description for the ACS.
+. Select {SmartProxies} to which the alternate content source is to be synced.
+. Enter the Base URL of the alternate content source.
+. Enter a comma-separated list of Subpaths.
+. Provide the *Manual Authentication* or *Content Authentication* credentials, if these are needed.
+. If SSL verification is required, enable *Verify SSL* and select the SSL CA certificate.
+. Review details and click *Add*.
+. Navigate to *Content* > *Alternate Content Sources* > click the vertical ellipsis next to the newly created alternate content source > Select *Refresh*.
+
+[id="cli-configuring-alternate-content-sources_{context}"]
+.CLI Procedure
+. On {ProjectServer}, enter the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# hammer alternate-content-sources create \
```

Comment:
```suggestion
# hammer alternate-content-source create \
```
(plus all occurrences bellow)

Also, we need to define the ACS type on creation time via `--alternate-content-source-type` argument. Possible value(s): 'custom', 'simplified', 'rhui'

Reply:
I'm not sure about the scope of this PR - does it refer to *Custom ACS* only or is it rather generic about ACS in general?
In the later case it might be good to mention the ACS types, their purpose and specifics.

Reply:
I think @AkshayGadhaveRH mentioned that is is only custom, and the RHUI + Simplified docs will come in another PR.

Reply:
Hey @vsedmik I was planning on making this chapter generic. Should I change the content to say that it is about custom ACS?
Also, as @ianballou pointed out, Simplified ACS will be in a separate PR which will follow this one.

Reply:
@AkshayGadhaveRH I just asked because the title says _Configuring Alternate Content Sources_, but the procedure describes the _Custom ACS_ creation only. So I felt we were missing the other types. If you are adding them in next PRs then perfect, we just need to update the hammer command.